src/app/replayer/sql.ml

```diff
@@ -218,6 +219,103 @@ module User_command = struct
     Conn.collect_list query user_cmd_id
 end
 
+module Snapp_command_ids = struct
+  let query =
+    Caqti_request.collect Caqti_type.string Caqti_type.int
+      (find_command_ids_query "snapp")
+
+  let run (module Conn : Caqti_async.CONNECTION) state_hash =
+    Conn.collect_list query state_hash
+end
+
+module Snapp_command = struct
+  type t =
+    { fee_payer_id : int
+    ; other_party_ids : int array
+    ; predicate_protocol_state_id : int
```

Comment:
Protocol state predicate has been removed from `Parties.t` It is now part of each party